covquestions-editor-app/src/store/questionnaireInEditor.ts

```diff
@@ -63,9 +63,22 @@ export const editTestCase = createAction<{
   hasErrors: boolean;
 }>("editTestCase");
 
+export const setDuplicatedIdInformation = createAction<{
```

Comment:
setDuplicatedIdInformation seems to be only dispatched, but never used in a reducer.

Reply:
Yes, you're right. That was part of the first idea, but implementation changed. I removed it.